src/Backend/Synapse/SynapseException.php

```diff
@@ -6,13 +6,49 @@
 
 use Doctrine\DBAL\Exception as DBALException;
 use Keboola\Db\Import\Exception;
+use Keboola\Db\ImportExport\Exception\ImportExportException;
+use Keboola\TableBackendUtils\Column\ColumnInterface;
 use Throwable;
 
-class SynapseException extends Exception
+class SynapseException extends ImportExportException
 {
+    public const CODE_TABLE_COLUMNS_MISMATCH = 1501;
```

Comment:
Nebylo by přínosné mít error code na type mismatch a na name mismatch?

Reply:
A nebylo by lepší ty konstanty mít u ImportExportException, aby byly na jednom místě?

Reply:
Plus mi tohle přijde jako ne-synapse-specifická věc...

Reply:
Nepřijde mě přínos mět to oddělené, je to to samé prostě ty tabulky musíja byt stejné a nejsů a je to specifické pro synapse žádný jiný backend to neřeší.

Reply:
Hlavně to žádný backend neřeší na téjto úrovni, tady je to user exception to pohnojil uživatel, ale pokud sa něco podobnéhos stane jinde je to náš internal, nemotal bych to dohromady.